graylog2-server/src/main/java/org/graylog2/bundles/BundleImporter.java

```diff
@@ -709,9 +729,19 @@ private void createLookupCaches(String bundleId, Set<LookupCacheBundle> lookupCa
                     .build());
             createdLookupCaches.put(dto.id(), dto);
         }
+
+        if (!createdLookupCaches.isEmpty()) {
+            clusterBus.post(CachesUpdated.create(createdLookupCaches.keySet()));
+        }
+
+        final Collection<LookupCache> caches = lookupTableService.getCaches(createdLookupCaches.keySet());
+        final CountDownLatch latch = new CountDownLatch(caches.size());
+        caches.forEach(c -> c.addListener(new LatchUpdaterListener(latch), scheduler));
+
+        latch.await(30, TimeUnit.SECONDS);
```

Comment:
We should at least log a short message on either INFO or WARN if the timeout expired (which means that not all caches were refreshed).